### DISCUSSION TITLE
[Suggestion] Monika's Voice: New Tips Found Using Statistics

### DISCUSSION THREAD

**Discussion by nopynospy** [Monika-After-Story/MonikaModDev#8239] [general]
Hello, I’m nopynospy. I saw the section, Monika’s “Voice”, at [this wiki page](https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines#monikas-voice), but since it’s a wiki section, it's not something that I can make changes to by forking.

I have done some basic statistical analysis on the original game scripts at my [repo](https://github.com/nopynospy/DokiDoki-Decompile).

Here are some insights I found in Monika’s lines:
•	Has the second highest average polarity (how positive is a statement), next to Sayori.
•	Has the highest average subjectivity (how subjective is a statement), meaning she tends to express opinions.
•	Has the highest average number of words per line (7).
•	Used almost twice as many ? (894) than ! (487).
•	The only girl that does not mention MC the most. It’s actually Sayori’s and Yuri’s names that occur the most, at 128 and 123. This is then followed by the MC, at 109. Natsuki only appeared 94 times.
•	“Ah”, “like”, “really”, “well”, “think”, “guess”, “yeah” and “right” are her most commonly used fillers.
•	“Poem”, “everyone”, “friends”, “share”, “read” and “write” are also among her 100 most used words.

**Comment by Dobby233Liu** [comment]
you can edit the page here https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines/_edit

**Comment by nopynospy** [comment]
Oh, cool, wasn't expecting it to be that easy. I mean, I thought there'd some sort of mechanisms to prevent trolls. Thanks, Liu.

**Comment by Booplicate** [comment]
To be honest, I'm not sure how this statistic can help with actual writing. It can be interesting, but it's not something people would look for when contributing.

**Comment by nopynospy** [comment]
Oh, okay. I know a bit of linguistics and literature, with some natural language processing. If I have new ideas in mind, would share them 🙂
